src/main/java/org/radarbase/appserver/service/scheduler/DataMessageSchedulerService.java

```diff
@@ -61,7 +61,7 @@ private static FcmDataMessage createMessageFromDataMessage(DataMessage dataMessa
                 .mutableContent(dataMessage.isMutableContent())
                 .deliveryReceiptRequested(IS_DELIVERY_RECEIPT_REQUESTED)
                 .messageId(String.valueOf(dataMessage.getFcmMessageId()))
-                .timeToLive(Objects.requireNonNullElse(dataMessage.getTtlSeconds(), 2_419_200))
+                .timeToLive(getValidTimeToLive(dataMessage.getTtlSeconds()))
```

Comment:
I would move this check to the FcmSender in the fcm package as this a FCM level restriction rather than of the appserver. This would ensure any future uses elsewhere of the send fcm function works as expected too.